dal-identification.yml

```diff
@@ -2955,23 +2955,23 @@ components:
         telephone:
           type: string
           maxLength: 10
-          pattern: '^\d{10}$'
+          #pattern: '^\d{10}$'
         extension:
           type: string
           maxLength: 7
-          pattern: '^\d{0,7}$'
+          #pattern: '^\d{0,7}$'
         cell:
           type: string
           maxLength: 10
-          pattern: '^\d{10}$'
+          #pattern: '^\d{10}$'
         tollFree:
           type: string
           maxLength: 10
-          pattern: '^\d{10}$'
+          #pattern: '^\d{10}$'
         fax:
           type: string
           maxLength: 10
-          pattern: '^\d{10}$'
+          #pattern: '^\d{10}$'
```

Comment:
a fax number should be only numbers and no string. why did we deactivate this validation rule?

Reply:
idem @franciskonchou